### PR TITLE
2023 11 06

### PR DESCRIPTION
**Pull request by lysackm**

### CODE REVIEW

File: documentation/meeting-logs/2023-11-13.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Standup Meeting 9
+# Standup Meeting 10
```
Comment (dlarocque) [inline documentation/meeting-logs/2023-11-13.md:1]: With this, our meeting numbers go from 7,8 to 10

**Review by dlarocque — APPROVED** [review]

File: documentation/meeting-logs/2023-11-06.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- Going to work on integration tests
+  - (https://github.com/BarbzCodez/Spendr/issues/109)
+
+Ethan:
+
+- Busy previous week
+
+Going over evaluation of other groups presentations
+
+Can move acceptance steps to a separate markdown file to go through when grading.
```
Comment (BarbzCodez) [inline documentation/meeting-logs/2023-11-06.md:58]: Just changed cleaned it up to be the same as the other ones

```suggestion
# Standup Meeting 8

2023-11-06
Sprint 3

Meeting duration: 1 hour

## Attendees

All group members

## Updates:

Mark:

- Edit and Delete budget endpoints created
  - [Modigying and Removing Budgets API #30](https://github.com/BarbzCodez/Spendr/issues/30)
- total expenses endpoint created
  - [Total Daily Expenses between Timeframes Endpoint #102](https://github.com/BarbzCodez/Spendr/issues/102)
- Some QA + bug fixing on Barbra's branch
- Cleaning up some documentation
- May take over Ethan's branch

Victoria:

- Finished the UI in analytics
  - [Spending Timeframe Comparison UI #37](https://github.com/BarbzCodez/Spendr/issues/37)
  - [Analytics Dashboard UI #36](https://github.com/BarbzCodez/Spendr/issues/36)
- Waiting for the endpoints
  - One in review
- Going to work on the homepage while the endpoints are being finished

Barbara:

- Group expenses, ui is done, hooking up is tricky
  - Hoping to finish today/tomorrow
  - [Group Expenses UI #41](https://github.com/BarbzCodez/Spendr/issues/41)
  - [Viewing and Managing Group Expenses #42](https://github.com/BarbzCodez/Spendr/issues/42)

Dan:

- created group expenses endpoints
  - [Fetch Group Expenses #120 ](https://github.com/BarbzCodez/Spendr/issues/120)
  - [Mark Group Expenses as Paid #40 ](https://github.com/BarbzCodez/Spendr/issues/40)
- Editing and deleting
  - going to leave it as a UI quirk
- Integration tests left to do
- Going to work on integration tests
  - [Integration Testing #109](https://github.com/BarbzCodez/Spendr/issues/109)

Ethan:

- Busy previous week

## Notes

Going over evaluation of other groups presentations

Can move acceptance steps to a separate markdown file to go through when grading.
```

File: documentation/meeting-logs/2023-11-13.md (hunk 1 above)
Comment (BarbzCodez) [reply documentation/meeting-logs/2023-11-13.md:1]: Just so its easier to commit

```suggestion
# Standup Meeting 9
```

**Review by BarbzCodez — COMMENTED** [review]
Little nitpicks

**Review by BarbzCodez — APPROVED** [review]

**Pull request merged**